Changelog.md

```diff
@@ -87,6 +87,7 @@ Compiler Features:
  * Tests: Determine transaction status during IPC calls.
  * Code Generator: Allocate and free local variables according to their scope.
  * Removed ``pragma experimental "v0.5.0";``.
+ * View Pure Checker: Warn about ``msg.value`` used by modifier in non-payable function.
```

Comment:
Perhaps `Warn about a modifier that contains ``msg.value`` being used in a non-payable function`?

Reply:
or 'applied to a non-payable function'?

Reply:
Ah, and this is not a compiler feature - I think it should be a breaking change.

Reply:
Oh, did I write that? :)